62_CAD/common.hlsl

```diff
@@ -197,6 +197,7 @@ NBL_CONSTEXPR uint32_t InvalidTextureIdx = nbl::hlsl::numeric_limits<uint32_t>::
 NBL_CONSTEXPR MajorAxis SelectedMajorAxis = MajorAxis::MAJOR_Y;
 // TODO: get automatic version working on HLSL
 NBL_CONSTEXPR MajorAxis SelectedMinorAxis = MajorAxis::MAJOR_X; //(MajorAxis) (1 - (uint32_t) SelectedMajorAxis);
+NBL_CONSTEXPR float MsdfPixelRange = 4.0;
```

Comment:
What is MsdfPixelRange?

Reply:
This is how many pixels, in MSDF resolution (32x32), of distance away (or inside) the shape that can be represented as a signed distance

(From msdfgen docs: specifies the width of the range around the shape between the minimum and maximum representable signed distance in shape units or distance field pixels, respectivelly.)

Reply:
oh okay I got it, because we're using a [0,1] range we need to map it somehow to [-MSDFRange/2, +MSDFRange/2]
and that gets you the distance in "texels" if you're inside the 32x32 image